packages/fe-mockserver-core/src/request/odataRequest.ts

```diff
@@ -128,6 +156,9 @@ export default class ODataRequest {
             this.applyDefinition.forEach((apply) => {
                 this._addSelectedPropertiesForApplyExpression(apply, additionalSelectProperty);
             });
+            for (const additionalSelectKey of Object.keys(additionalSelectProperty)) {
+                addPathToExpandParameters(additionalSelectKey, this.expandProperties, undefined, true, false);
```

Comment:
I wonder whether it should be `..., true)` instead - to remove the expansion from the result 🤔